src/project/LoginView.java

```diff
@@ -0,0 +1,90 @@
+package project;
+
+import javafx.application.Application;
+import javafx.scene.control.ComboBox;
+import javafx.collections.ObservableList;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+import javafx.scene.Scene;
+import javafx.scene.text.Text;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextField;
+import javafx.scene.control.Button;
+import javafx.scene.layout.FlowPane;
+import javafx.geometry.Pos;
+import javafx.event.ActionEvent;
+
+public class Login extends Application{
```

Comment:
`public class LoginView extends Application{`
Just in case we want to use pass this to a LoginController class